Command/TranslationUpdateCommand.php

```diff
@@ -71,7 +69,16 @@ protected function execute(InputInterface $input, OutputInterface $output)
 
         $bundleTransPath = $bundle->getPath().'/Resources/translations';
 
-        $loader->loadMessages($bundleTransPath, $catalogue);
+        if ($this->getContainer()->has('translation.reader')) {

```

Comment:
Please introduce an explanatory local variable, i.e:

`$isSf4 = $this->getContainer()->has('translation.reader');`

Now you can use it in the conditional statement. Without a local variable one only can guess why this conditional statements are added here.